packages/template-retail-react-app/app/components/store-locator/store-locator-content.jsx

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React from 'react'
+import {useIntl} from 'react-intl'
+import PropTypes from 'prop-types'
+import {
+    Heading,
+    Button,
+    InputGroup,
+    Accordion,
+    AccordionItem,
+    Select,
+    Box,
+    Input
+} from '@salesforce/retail-react-app/app/components/shared/ui'
+import StoresList from '@salesforce/retail-react-app/app/components/store-locator/stores-list'
+import {Controller, useForm} from 'react-hook-form'
+import {SUPPORTED_STORE_LOCATOR_COUNTRIES} from '@salesforce/retail-react-app/app/constants'
+
+const StoreLocatorContent = (props) => {
```

Comment:
Would you consider extracting the Store Locator Form (eg. All of the inputs) into a seperate component so it could be more easily overridden?